src/pages/booking.html

```diff
@@ -7,4 +7,4 @@
 content_blocks:
 ---
 
-<h1>booking</h1>
\ No newline at end of file
+<h1>booking</h1>
```

Comment:
What is the purpose of this change?

Reply:
this was just code from the boiler plate, looks like a commit removed it and then added back? These lines on each page can be deleted completely.